pkg/build/strategies/layered/layered.go

```diff
@@ -63,14 +64,24 @@ func (b *Layered) CreateDockerfile(config *api.Config) error {
 	}
 
 	buffer.WriteString(fmt.Sprintf("FROM %s\n", b.config.BuilderImage))
-	buffer.WriteString(fmt.Sprintf("COPY scripts %s\n", locations[0]))
+	// if -s / --scripts / --scripts-url are not specified, there will not be a scripts subdir
+	// off of the working dir, so don't copy here, and we'll have to tweak the scripts URL to use the env var
+	// STI_SCRIPTS_PATH
+	scriptsIncluded := len(config.ScriptsURL) > 0
```

Comment:
@bparees - testing could be warranted, but when I look at https://github.com/openshift/source-to-image/blob/master/pkg/scripts/install.go#L113 and https://github.com/openshift/source-to-image/blob/master/pkg/scripts/download.go#L54-L63, I believe the result is that based on the `image` preface to the url, it will essentially copy from the image destination "scripts" directory that will be in the tar, so we would be ok there.  Or am I misreading this?

@mfojtik - just want to make sure I understand you:  you mean changing the `COPY` statement currently generated in the layered Dockerfile to a `RUN` that has a bash snippet that tests ./scripts existence before doing a cp, correct ?  If so, I can tell you I did the equivalent of that already, and it wasn't sufficient - we needed the conditional setting of ScriptsURL below based on the same conditional, which we would not have if done in the docker file.
If you mean checking literally for the presence of /tmp/..../scripts in the go code, I think the scripts dir is there when it isn't in the builder image (though not as sure there, and can double check if this is the path we want to go down.